src/tigerbeetle.zig

```diff
@@ -131,49 +133,42 @@ pub const CreateTransferResult = enum(u32) {
     ok,
     linked_event_failed,
     exists,
+    exists_user_data,
     exists_with_different_debit_account_id,
     exists_with_different_credit_account_id,
     exists_with_different_user_data,
+    exists_with_different_ledger,
     exists_with_different_reserved_field,
     exists_with_different_code,
     exists_with_different_amount,
     exists_with_different_timeout,
     exists_with_different_flags,
-    exists_and_already_committed_and_accepted,
-    exists_and_already_committed_and_rejected,
     reserved_field,
     reserved_flag_padding,
     debit_account_not_found,
     credit_account_not_found,
     accounts_are_the_same,
-    accounts_have_different_units,
+    accounts_have_different_ledgers,
     amount_is_zero,
     exceeds_credits,
     exceeds_debits,
-    two_phase_commit_must_timeout,
-    timeout_reserved_for_two_phase_commit,
-};
-
-pub const CommitTransferResult = enum(u32) {
-    ok,
-    linked_event_failed,
-    reserved_field,
-    reserved_flag_padding,
+    pending_transfer_must_timeout,
+    timeout_reserved_for_pending_transfer,
+    // For two-phase transfers:
+    cannot_post_and_void_pending_transfer,
     transfer_not_found,
-    transfer_not_two_phase_commit,
+    transfer_not_pending,
+    transfer_already_posted,
+    transfer_already_voided,
     transfer_expired,
-    already_committed,
-    already_committed_but_accepted,
-    already_committed_but_rejected,
-    debit_account_not_found,
-    credit_account_not_found,
-    debit_amount_was_not_reserved,
-    credit_amount_was_not_reserved,
-    exceeds_credits,
-    exceeds_debits,
     condition_requires_preimage,
-    preimage_requires_condition,
     preimage_invalid,
+    preimage_requires_condition,
```

Comment:
We should drop any errors relating to `condition` and `preimage` now, since we've dropped hash locks for the time being.

Reply:
Done.